.github/workflows/main.yml

```diff
@@ -0,0 +1,123 @@
+name: CI Lint and Tests
+
+# Controls when the action will run. Triggers the workflow on pull request
+# events or push events in the develop branch.
+on: 
+  pull_request:
+  push:
+    branches:    
+      # Push events on develop branch
+      - develop
+
+# This workflow has the following jobs:
+# linters: Kotlin and Protobuf Linting
+# robolectric_tests: Robolectric tests for all modules except the app module
+# app_tests: Non-flaky Robolectric tests for the app module
+jobs:
+  linters:
+    name: Lint Tests
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Download Ktlint
+      run: |
+        KTLINT="0.37.0"
+        echo Using Ktlint $KTLINT
+        curl -sSLO https://github.com/pinterest/ktlint/releases/download/$KTLINT/ktlint
+        chmod a+x ktlint
+    
+    -  name: Download buf
+       run: |
+        BUFVERSION="v0.15.0"
+        echo Using Bufbuild version $BUFVERSION
+        curl -sSL https://github.com/bufbuild/buf/releases/download/$BUFVERSION/buf-$(uname -s)-$(uname -m) > buf
+        chmod a+x buf
+        
+    - name: Kotlin lint check
```

Comment:
LGTM
I think afterwards we can update the name as before like what modules we have in ktlint check `Data, Utility, Domain`

Reply:
Sounds good!